### PR TITLE
duplicate: fix spelling error

### PR DESCRIPTION
**Pull request by dtorop**
Fix tooltip spelling error

### CODE REVIEW

**Comment by dtorop** [comment]
Calling images "virgin" is a bit loaded. I made another commit with simpler language.

File: src/libs/duplicate.c [hunk 1]
```diff
@@ -353,11 +353,11 @@ void gui_init(dt_lib_module_t *self)
   GtkWidget *bt = gtk_label_new(_("existing duplicates"));
   gtk_box_pack_start(GTK_BOX(hb), bt, FALSE, FALSE, 0);
   bt = dtgtk_button_new(dtgtk_cairo_paint_plusminus, CPF_ACTIVE | CPF_STYLE_FLAT | CPF_DO_NOT_USE_BORDER, NULL);
-  g_object_set(G_OBJECT(bt), "tooltip-text", _("create a 'virgin' duplicate of the image without any developpement"), (char *)NULL);
+  g_object_set(G_OBJECT(bt), "tooltip-text", _("duplicate image without its history"), (char *)NULL);
```
Comment (TurboGit) [inline src/libs/duplicate.c]: What about:
"duplicate image without its history" -> "duplicate image with pristine history stack"

Comment (dtorop) [reply src/libs/duplicate.c]: That sounds good to me. I'll update this...

**Comment by TurboGit** [comment]
was fixed manually for 2.6.

**Pull request closed without merging**